### PR TITLE
[cli] Add `mcap decompress` command

### PR DESCRIPTION
**Pull request by jhurliman**
**Public-Facing Changes**
- [cli] `mcap decompress` command, alias for `mcap compress --compression "none"`

### CODE REVIEW

File: go/cli/mcap/cmd/filter.go [hunk 1]
```diff
@@ -64,6 +64,7 @@ func buildFilterOptions(flags filterFlags) (*filterOpts, error) {
 	case "lz4":
 		opts.compressionFormat = mcap.CompressionLZ4
 	case "none":
+	case "":
```
Comment (james-rms) [inline go/cli/mcap/cmd/filter.go:67]: why this change? i figure it's clearer to require a non-empty string if `--compression` is supplied.

**Review by james-rms — COMMENTED** [review]
I don't really see the use-case, but if you have a use for it 👍

File: go/cli/mcap/cmd/filter.go (hunk 1 above)
Comment (jhurliman) [reply go/cli/mcap/cmd/filter.go:67]: I tried to run the command `mcap compress --compression "" input.mcap -o output.mcap` and it failed, telling me to use `"none"` instead. Not a terrible outcome (it told me what to do), but why not just do what I asked and use the `""` compression mode as defined in the MCAP spec?
